core.h

```diff
@@ -0,0 +1,284 @@
+#include <stdint.h>
```

Comment:
Add the following code in the beginning of this header:
```cpp
#ifndef GAMEBOYEMU_CORE
#define GAMEBOYEMU_CORE
```

In the end of this header, of course, add the corresponding line:
```cpp
#endif
```